docs/source/morpheus_quickstart_guide.md

```diff
@@ -0,0 +1,1264 @@
+  in the order that data should flow. The output of each stage will become the
+  input for the next stage. For example, to read, classify and write to a
+  file, the following stages could be used
+
+  pipeline from-file --filename=my_dataset.json deserialize preprocess inf-triton --model_name=my_model
+  --server_url=localhost:8001 filter --threshold=0.5 to-file --filename=classifications.json
+
+  Pipelines must follow a few rules:
+  1. Data must originate in a source stage. Current options are `from-file` or `from-kafka`
+  2. A `deserialize` stage must be placed between the source stages and the rest of the pipeline
+  3. Only one inference stage can be used. Zero is also fine
+  4. The following stages must come after an inference stage: `add-class`, `filter`, `gen-viz`
+
+Options:
+  --columns_file FILE        [default: data/columns_ae.txt]
+  --labels_file FILE         Specifies a file to read labels from in order to
+                             convert class IDs into labels. A label file is a
+                             simple text file where each line corresponds to a
+                             label. If unspecified, only a single output label
+                             is created for FIL
+  --userid_column_name TEXT  Which column to use as the User ID.  [default:
+                             userIdentityaccountId; required]
+  --userid_filter TEXT       Specifying this value will filter all incoming
+                             data to only use rows with matching User IDs.
+                             Which column is used for the User ID is specified
+                             by `userid_column_name`
+  --viz_file FILE            Save a visualization of the pipeline at the
+                             specified location
+  --help                     Show this message and exit.
+
+Commands:
+  add-class        Add detected classifications to each message
+  add-scores       Add probability scores to each message
+  buffer           (Deprecated) Buffer results
+  delay            (Deprecated) Delay results for a certain duration
+  filter           Filter message by a classification threshold
+  from-cloudtrail  Load messages from a Cloudtrail directory
+  gen-viz          (Deprecated) Write out vizualization data frames
+  inf-pytorch      Perform inference with PyTorch
+  inf-triton       Perform inference with Triton
+  monitor          Display throughput numbers at a specific point in the
+                   pipeline
+  preprocess       Convert messages to tokens
+  serialize        Include & exclude columns from messages
+  timeseries       Perform time series anomaly detection and add prediction.
+  to-file          Write all messages to a file
+  to-kafka         Write all messages to a Kafka cluster
+  train-ae         Deserialize source data from JSON
+  validate         Validates pipeline output against an expected output
+```
+
+### ONNX to TRT Conversion
```

Comment:
Deferred per today's discussion

Reply:
well, @cwharris has a check teed up in #23 
so not sure what we want to say here
pip install ...?

Reply:
Has this been fixed?

Reply:
Removed ONNX to TRT for now.